app/src/app/components/sidebar/MapModeSelector.jsx

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+import * as RadioGroup from "@radix-ui/react-radio-group";
+import { styled } from "@stitches/react";
+import { blackA } from "@radix-ui/colors";
+import { useMapStore } from "../../store/mapStore";
+import { RadioCards, Box } from "@radix-ui/themes";
+
+export function MapModeSelector() {
+  const mapStore = useMapStore.getState();
+  const { activeTool, setActiveTool } = useMapStore((state) => ({
+    activeTool: state.activeTool,
+    setActiveTool: state.setActiveTool,
+  }));
+
+  if (!activeTool) return null;
+  const activeTools = [
+    { mode: "pan", disabled: false, label: "Pan" },
+    { mode: "brush", disabled: false, label: "Brush" },
+    { mode: "erase", disabled: true, label: "Erase" },
+  ];
+
+  const handleRadioChange = (value) => {
+    console.log("setting active tool to", value);
+    setActiveTool(value);
+  };
+
+  return (
+    <Box>
+      <RadioCards.Root
+        defaultValue="default"
+        value={activeTool}
+        onValueChange={handleRadioChange}
+        columns={{ initial: "1", sm: "3" }}
+      >
+        {activeTools.map((tool) => (
+          <Flex key={`${tool.mode}-flex`} direction="column" width="100%">
+            <RadioCards.Item
+              value={tool.mode}
+              id={tool.mode}
+              disabled={tool.disabled}
+            >
+              {tool.label}
```

Comment:
O: Low effort nice to have would be adding some [icons](https://www.radix-ui.com/icons)

Reply:
added!